test/Transforms/PolyLowering/poly_lowering_fail_reserved_name.llzk

```diff
@@ -0,0 +1,22 @@
+// RUN: llzk-opt -I %S -split-input-file -llzk-full-poly-lowering="max-degree=2" -verify-diagnostics %s
+
+module attributes {veridise.lang = "llzk"} {
+  struct.def @CmpConstraint {
+    // expected-error@+1 {{Field name: "__llzk_aux_field_0" starts with prefix: __llzk_aux_field_ which is reserved for lowering pass}}
```

Comment:
Recommend putting quotes around `__llzk_aux_field_` as well

Reply:
Also, recommend specifying what kind of lowering pass b/c that's a general term in MLIR.